packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -0,0 +1,210 @@
+%global nvidia_tesla_470_version 470.82.01
+%global spdx_id %(bottlerocket-license-tool -l $PWD/rpmbuild/BUILD/Licenses.toml spdx-id nvidia)
+%global license_file %(bottlerocket-license-tool -l $PWD/rpmbuild/BUILD/Licenses.toml path nvidia -p ./licenses)
+
+Name: %{_cross_os}kmod-5.10-nvidia
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: NVIDIA drivers for the 5.10 kernel
+# We use these licences because we only ship our own software in the main package,
+# each subpackage includes the LICENSE file provided by the run scripts
```

Comment:
I need to fix/remove this line, to explain that subpackages include the license provided by the `Licenses.toml` configuration